frontend/src/screens/settings/AutoUnlock.tsx

```diff
@@ -82,9 +81,9 @@ export function AutoUnlock() {
           <>
             <form
               onSubmit={onSubmit}
-              className="w-full flex flex-col gap-3 mt-3"
+              className="w-full md:w-[373px] flex flex-col gap-4 mt-4"
```

Comment:
please avoid hardcoding pixel values, why is this needed?

Reply:
![image](https://github.com/user-attachments/assets/68ab8db4-c84e-476a-a67d-21da8b73a982)

i see hardcoded values in manyplaces so i followed exact designs given by jakub. thats for unlock password. all the inputs are of 373px in width as per new designs. next possible width can be w-96 (384px) if you don't want hardcoded values

![image](https://github.com/user-attachments/assets/a54e277b-a7ed-42d5-942f-223cd7ac2c55)

Reply:
@stackingsaunter can you also try to use tailwind units as much as possible?